starter/script.js

```diff
@@ -1 +1,108 @@
 // put code here!
+function ultimatelyGreedyDragon(){
+  document.write("Unfortunately, the treasures are cursed. Any one adventurer who attempts to obtain treasure from\nboth areas in the cave loses touch with reality, becomes lost in the cave and ultimately goes insane.\nYou head toward the spider cave but suddenly forget where you were going and why. You only\nknow that you feel a deep sense of dissatisfaction. You feel you should be searching for something, so\nyou wander the maze aimlessly. Lonely and unhappy, you begin to mumble to yourself. You feel the words\nbegin to lose their meaning over time until you're sure you are just speaking gibberish, but you continue anyway\nso you can hear your own voice to be sure you're still alive. GAME OVER. YOU LOSE.")
+}
+function notGreedyReason(){
+  document.write("You have chosen to leave the cave. As you take a step toward the exit, a benevolent voice booms in your head.\n'You have chosen well, adventurer. The third treasure is granted. You will keep your sanity and your treasure.\nYou will also have a heart of gold. But not in a literal sense because that would kill you. You reasoned with\nthe dragon and lived, and you were not greedy. Thus you will be able to navigate social situations with ease.\nDue to your lack of greed and you social skills, you will have strong relationships, and a truly valuable life.'\nGAME OVER. YOU WIN.");
+
+}
+
+var questResponse = prompt("You are an adventurer. You have travelled to a cave high in a mountain range.\nLegend has it that there are three treasures within the cave. One treasure is guarded by a dragon.\nOne treasure is hidden in a so called spider-maze. Nothing is known about the third treasure.\nYou have an old map which shows the location of the dragon and the start of the maze.\nYou don't have any information about the third treasure. Would you like to head for the 'dragon' or the 'maze'?");
+if (questResponse == "dragon" || questResponse == 'd' || questResponse == 'drgn' || questResponse.toLowerCase().indexOf('d') == 0) {
```

Comment:
instead of troubling yourself too much about what people can respond, for prototyping just assume the user will use the interface correctly.